app/controllers/api/v1/api_docs_controller.rb

```diff
@@ -111,6 +111,14 @@ class ApiDocsController < ActionController::Base
       def index
         render json: Swagger::Blocks.build_root_json(SWAGGERED_CLASSES)
       end
+
+      def swagger_ui
+
+      end
+
+      def oauth2_redirect
+
+      end
```

Comment:
Can these empty methods be removed?

```suggestion
```

Reply:
No - they are required as per standard Rails routing.  In order for a Rails server to respond to a request, there are three prerequisites.

- An entry in `routes.rb` describing the url, method, and mapping to controller/action
- A corresponding method declaration in the aforementioned controller
- Either a corresponding view template, or a `render` call in the controller action describing what to render